content/community/resources/bees/contents.lr

```diff
@@ -0,0 +1,19 @@
+title: ...Bees?
+---
+author: Katie McLaughlin
+---
+description:
+
+Cross-platform application development is a holy grail of software engineering. Write once, run everywhere - desktops, mobiles, and the web. There have been many attempts at this over the years, but the absolute pinnacle of this art form is to have no-one notice - you want your apps to look and feel like native apps so that your users can't tell. Bonus points is if your development happens in a language you already know and use. This is the goal of BeeWare: a suite of application tools and libraries that to develop native cross platform applications in Python. During this short presentation Katie will take you on a tour of the BeeWare stack, and describe how we've used the project as an incubator for new open source contributors around the world.
+
+(The talk was unfortunately not recorded, but Katie has recorded a screencast for us!)
+---
+event_name: GitHub Constellation Sydney
+---
+event_url: githubconstellation.com/sydney/
```

Comment:
@glasnt could you please change this to 

http://githubconstellation.com/sydney/